source/_posts/Applying-Proguard-in-an-android-application.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Applying Proguard in an android application
```

Comment:
Title   tag character should be 50-60 character

Reply:
Updated